add multi-modal-docs

hwchase17 opened the pull request:
We dont really have any abstractions around multi-modal... so add a section explaining we dont have any abstrations and then how to guides for openai and anthropic (probably need to add for more)

hwchase17 committed:
add multi-modal-docs

hwchase17 committed:
cr

ccurme committed:
format

ccurme committed:
add cell on tool calling

ccurme commented on docs/docs/how_to/multi-modal-inputs.ipynb:
```diff
@@ -0,0 +1,228 @@
+   "id": "71bd28cf-d76c-44e2-a55e-c5f265db986e",
+   "metadata": {},
+   "source": [
+    "## Tool calls\n",
```
added a cell on tool calling so we can link to it if people ask. feel free to modify or remove.

hwchase17 committed:
Merge branch 'master' into harrison/multi-modal-docs

tomasonjo committed:
community[patch]: Better error message for neo4j vector when text is null (#21861)

eyurtsev committed:
docs: run migration script against how-to docs (#21927)

Upgrade imports in how-to docs

eyurtsev committed:
docs: migrate tutorials using langchain-cli migrate (#21928)

Migrate tutorials

eyurtsev committed:
docs: migrate integrations using langchain-cli (#21929)

Migrate integration docs

junefish committed:
docs: update notebook for new Pinecone API + serverless (#21923)

Thank you for contributing to LangChain!

- [x] **PR title**: "docs: update notebook for new Pinecone API +
serverless"


- [x] **PR message**: The published notebook is not runnable after
`pinecone-client` v2, which is deprecated. `langchain-pinecone` is not
compatible with the latest `pinecone-client` (v4), so I hardcoded it to
the last v3. Also updated for serverless indexes (only index type
available on Pinecone free plan).


- [x] **Add tests and docs**: N/A (tested in Colab)


- [x] **Lint and test**: Run `make format`, `make lint` and `make test`
from the root of the package(s) you've modified. See contribution
guidelines for more: https://python.langchain.com/docs/contributing/

Additional guidelines:
- Make sure optional dependencies are imported within a function.
- Please do not add dependencies to pyproject.toml files (even optional
ones) unless they are required for unit tests.
- Most PRs should not touch more than one package.
- Changes should be backwards compatible.
- If you are adding something to community, do not re-import it in
langchain.

If no one reviews your PR within a few days, please @-mention one of
baskaryan, efriis, eyurtsev, hwchase17.


---
- To see the specific tasks where the Asana app for GitHub is being
used, see below:
  - https://app.asana.com/0/0/1207328087952500

ccurme committed:
mistral: implement ls_params (#21867)

junefish committed:
docs: update notebook for latest Pinecone API + serverless (#21921)

Thank you for contributing to LangChain!

- [x] **PR title**: "docs: update notebook for latest Pinecone API +
serverless"


- [x] **PR message**: Published notebook is incompatible with latest
`pinecone-client` and not runnable. Updated for use with latest Pinecone
Python SDK. Also updated to be compatible with serverless indexes (only
index type available on Pinecone free tier).


- [x] **Add tests and docs**: N/A (tested in Colab)


- [x] **Lint and test**: Run `make format`, `make lint` and `make test`
from the root of the package(s) you've modified. See contribution
guidelines for more: https://python.langchain.com/docs/contributing/

Additional guidelines:
- Make sure optional dependencies are imported within a function.
- Please do not add dependencies to pyproject.toml files (even optional
ones) unless they are required for unit tests.
- Most PRs should not touch more than one package.
- Changes should be backwards compatible.
- If you are adding something to community, do not re-import it in
langchain.

If no one reviews your PR within a few days, please @-mention one of
baskaryan, efriis, eyurtsev, hwchase17.


---
- To see the specific tasks where the Asana app for GitHub is being
used, see below:
  - https://app.asana.com/0/0/1207328087952499

ccurme committed:
partners: bump core in packages implementing ls_params (#21868)

These packages all import `LangSmithParams` which was released in
langchain-core==0.2.0.

N.B. we will need to release `openai` and then bump `langchain-openai`
in `together` and `upstage`.

ccurme committed:
update maintainers (#21305)

hinthornw committed:
rm hyphen

hinthornw committed:
Update language

hinthornw committed:
spell

baskaryan committed:
fmt

baskaryan committed:
fmt

baskaryan committed:
fmt
